qml/StatisticsChartView.qml

```diff
@@ -23,33 +23,36 @@ ChartView {
     antialiasing: true
     legend.visible: false
 
+    margins.bottom: 0
+    margins.left: 0
+    margins.right: 10
+    margins.top: 0
+
     property var mapper: []
     property var axisYItem: axisY
     property var dateTimeAxisXItem: dateTimeAxisX
     property var tooltipItem: tooltip
 
-    property int axisYMin: 0
-    property int axisYMax: 10
-    property date dateTimeAxisXMin: new Date()
-    property date dateTimeAxisXMax: new Date()
-
     signal clearedChart()
 
+    Component.onCompleted: {
+        axisY.applyNiceNumbers()
+    }
+
     ValueAxis {
         id: axisY
-        min: axisYMin
-        max: axisYMax
+        min: 0
+        max: 1
```

Comment:
Fix the number of labels to 5.

```suggestion
        max: 1
        tickCount: 5
```

Reply:
Set also:
```
minorTickCount: 5
onTickCountChanged: tickCount = 5
```

Reply:
with `fineNumbers` is automaticly fit to round numbers, we may want to leave it this way. (I change it anyway).

Reply:
This looks like it does not work with fineNumbers